dll/appcompat/apphelp/sdbapi.c

```diff
@@ -347,6 +347,21 @@ PDB WINAPI SdbOpenDatabase(LPCWSTR path, PATH_TYPE type)
     {
         SHIM_INFO("Failed to get the database id\n");
     }
+
+    root = SdbFindFirstTag(pdb, TAGID_ROOT, TAG_DATABASE);
+    if(root != TAGID_NULL)
```

Comment:
```suggestion
    if (root != TAGID_NULL)
```

Reply:
I wont commit this with the web interface cause that'll give me local conflicts.